### PR TITLE
Restrict Domain and NdArray dimensionality to >= 1

### PR DESCRIPTION
**Pull request by jonblower**
Addresses #68

### CODE REVIEW

**Comment by jonblower** [comment]
@letmaik does this look OK?

File: standard_template/standard/clause_specification_text.adoc [hunk 1]
```diff
@@ -574,12 +575,12 @@ Example of a domain object with [`"Trajectory"`][domain-types] domain type:
 
 ### 6.2. NdArray Objects
 
-A CoverageJSON object with the type `"NdArray"` is an NdArray object. It represents a multidimensional (>= 0D) array with named axes, encoded as a flat one-dimensional array in row-major order.
+A CoverageJSON object with the type `"NdArray"` is an NdArray object. It represents a multidimensional (>= 1D) array with named axes, encoded as a flat one-dimensional array in row-major order.
 
 - An NdArray object MUST have a member with the name `"values"` where the value is an array of numbers and nulls, or strings and nulls, where nulls represent missing data.
```
Comment (letmaik) [inline standard_template/standard/clause_specification_text.adoc:580]:
```suggestion
- An NdArray object MUST have a member with the name `"values"` where the value is a non-empty array of numbers and nulls, or strings and nulls, where nulls represent missing data.
```

While the domain axes wouldn't allow empty ranges it makes sense to forbid empty arrays completely at this layer.

**Review by letmaik — COMMENTED** [review]
Looks good, just one minor addition.

File: standard_template/standard/clause_specification_text.adoc (hunk 1 above)
Comment (jonblower) [reply standard_template/standard/clause_specification_text.adoc]: Agreed and done in latest commit

**Review by chris-little — APPROVED** [review]
@letmaik @jonblower Looks good to me. We can always loosen back to 0D if a convincing use case emerges.

**Comment by chris-little** [comment]
Agreed at 2022-05-11 meeting to merge and close issue

**Pull request merged**